rifServices/src/main/java/org/sahsu/rif/services/datastorage/common/BaseSQLManager.java

```diff
@@ -359,6 +501,29 @@ else if (databaseType == DatabaseType.SQL_SERVER) {
 		return columnComment;
 	}
 	
+	/** 
```

Comment:
Consider using a library for this. For example [this SO question](https://stackoverflow.com/questions/25680258/converting-a-string-from-snake-case-to-camel-case-in-java) suggests that Guava can do it, and we already use that.

Reply:
Done